### PR TITLE
Migrate tap-mongodb menzenski variant to MeltanoLabs

### PR DESCRIPTION
**Pull request by menzenski**
Discussion https://github.com/MeltanoLabs/Meta/issues/44

CC @kgpayne

### CODE REVIEW

**Review by pnadolny13 — APPROVED** [review]
@menzenski thanks for this! Looks great.

We'll just have to wait for the repo to transfer before we merge this.

**Comment by menzenski** [comment]
@pnadolny13 see my [comment](https://github.com/MeltanoLabs/Meta/issues/44#issuecomment-1639347358) on the other issue - I was unable to initiate the transfer.

**Comment by menzenski** [comment]
@pnadolny13 the repository has been transferred! https://github.com/MeltanoLabs/tap-mongodb

Thank you @WillDaSilva for the help

**Pull request merged**
